tests/foreman/ui/test_host.py

```diff
@@ -1811,9 +1811,14 @@ def test_positive_delete_libvirt(
 
 
 @pytest.fixture
-def gce_client(download_cert):
+def gce_cert():
+    return json.loads(settings.gce.cert)
+
+
+@pytest.fixture
+def gce_client(download_cert, gce_cert):
```

Comment:
Even this fixture is duplicated in multiple test modules. We can create a new module in `pytest_fixtures` for GCE fixtures, or we can slot this into sys_fixtures?